### PR TITLE
Create composer.json

### PR DESCRIPTION
**Pull request by JelmerD**
For composer compatibility (easier to install)

### CODE REVIEW

**Comment by JelmerD** [comment]
Oops, I made a mistake.

**Comment by FMCorz** [comment]
Many thanks for this patch. Can I just ask 2 minor things?
- Is it a standard to use - instead of camelcase? I find it a bit confusing that the name of the repo differs from the composer name.
- Could you add a brief instruction on how to install the package using composer in the readme?

Many thanks!
Fred

**Comment by JelmerD** [comment]
1. Yes, the lowercase is necessary for compatibility with https://packagist.org/. I will write you a message on how to set this up properly. But I will do that once it is merged, otherwise it won't work at all :)
   For an example: https://packagist.org/packages/jelmerd/log-watcher
   The lowercase is used to match the url, the url is always lowercase at packagist, and it has to match that url. What you can do already is register yourself at packagist.org (login with Github is easiest). Packagist is making your package available for composer (since composer only has a few domains it checks for packages).
2. I will update the readme.md with a small instruction (I think most people who used composer before know what to do though).

**Comment by JelmerD** [comment]
Done. I also updated your code example to improve readability and added php syntax highlighting.

**Comment by ErikvdVen** [comment]
:+1:

**Comment by FMCorz** [comment]
Thanks. I've squashes your commits and tweaked the readme file.

New version available, many thanks!

https://github.com/FMCorz/AclManager/compare/ee1c569...1.2.5

**Comment by JelmerD** [comment]
You're welcome. Okay, now.
- Go to https://packagist.org/packages/submit
- Login with Github
- Add https://github.com/FMCorz/AclManager/

### then
- Navigate to https://github.com/FMCorz/AclManager/settings/hooks
- Select `packagist` from the list at the left
- Fill in the form with the values provided in https://packagist.org/profile/

Then, when you push something to your `stable` branch, it should update the packagist. Since composer won't look at repositories at github, it will only search on certain domains, like packagist.org for a matching composer file. So you have to add it there in order for it to work.

I can add your package as well, but it's your project, so it's better when you do it. That will guarantee full control on your side :)

**Comment by FMCorz** [comment]
Done! Thanks.

**Comment by JelmerD** [comment]
Well, it is working perfectly. Now I can install this easier on my projects. As you can see in my repo I once started a CakeAcl plugin, but I think I will help make your plugin better. There are some functionalities I am missing at this point. Main point: JavaScript with Ajax calls. I hate it to reload the page every time I change a setting. It would be nice have some sort of instant update :) But I guess we can work on that ;-)

Thanks for putting in the effort to make the composer installing work.

**Comment by FMCorz** [comment]
Thanks to you!

There has been a pull request to use JQuery, it might be a start. Though I would like to see something that uses the JS Helpers of Cake rather than hardcoding dependencies on X or Y JS engine.

**Comment by JelmerD** [comment]
I have never used the JsHelper, just because I like to be in control of the JavaScript more. I like to just have a .js file where I can put in my code. So that I can have JavaScript autocompletion and have control over my variables just a little bit more. Also, taking out this extra helper should improve loading times.

TL;DR:
I wouldn't use the JsHelper.
